setup.py

```diff
@@ -142,6 +142,7 @@
     'rb>=1.7.0,<2.0.0',
     'qrcode>=5.2.2,<6.0.0',
     'python-u2flib-server>=4.0.1,<4.1.0',
+    'redis-py-cluster>=1.3.4',
```

Comment:
Are we committing to this being a universally required dependency out of the gate (e.g. for on premise?)

I guess there's probably no harm in it … ?

Reply:
I don't see a problem with it. But I would recommend capping the max version. Too many problems with dependency versions lately.

Reply:
d215848e885ae7b5243a46206f043cb73929f238